### PR TITLE
WIP: New package: qjoypad-4.3.1

### PR DESCRIPTION
**Pull request by lilmike**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/qjoypad/template [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Template file for 'qjoypad'
+pkgname=qjoypad
+version=4.3.1
+revision=1
+#wrksrc=
```
Comment (abenson) [inline srcpkgs/qjoypad/template:5]: Drop the comment lines

File: srcpkgs/qjoypad/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+#create_wrksrc=yes
+build_style=cmake
+#configure_args=""
+#make_build_args=""
+#make_install_args=""
+#conf_files=""
+#make_dirs="/var/log/dir 0755 root root"
+hostmakedepends="qt5-tools cmake pkg-config"
+makedepends="qt5-devel libXtst-devel qt5-x11extras-devel qt5-tools-devel"
+depends="qt5 libXtst qt5-x11extras"
```
Comment (abenson) [inline srcpkgs/qjoypad/template:15]: This should be automatically detected by the build system.

**Comment by lilmike** [comment]
Sorry about that, forgot about that one. Fixed.
-Michael.

-- 
Michael Taboada, Creator of Games. Does that make me a god, well, sorta.
My pgp key: 79BC390E8864CD71305D4DD606CD2197A6EE5ED7
My website: https://michaels.world
2MB website: https://2mb.games
Don't believe everything you read on the internet. I might not be real, you might not be real, and this email certainly isn't real.
Sent from my newfangled...
stationary bicycle

**Comment by lilmike** [comment]
>This should be automatically detected by the build system.

Which should. Depends or the commented values?

**Comment by lilmike** [comment]
I'm marking this as WIP, because upon fixing the arm issues with cmake, there are actual syntax errors which prevent the package from building -- but only on arm. I've opened an issue upstream and hope to have a fix soon.
-Michael.

**Comment by abenson** [comment]
> Depends or the commented values?

It'll automatically detect the depends in most cases

**Comment by benalb** [comment]
`post_extract() {
        vsed -i CMakeLists.txt -e "s; -Werror;;"
}
`
This should fix the [error](https://github.com/panzi/qjoypad/issues/48)
builds ok, and I've tested it in x86_64.

**Pull request closed without merging**